### PR TITLE
[UXE-2474] feat: new additional data

### PR DESCRIPTION
**Pull request by william-tome**
### What is the new behavior introduced by this PR?
- create new additional data layout
- implement tracking events
- integrate with v4 API
- create tests

### Does this PR introduce breaking changes?
- [ ] No
- [x] Yes 

Implement new view with different fields and API contract

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/95643165/e57e4b1a-fced-4166-a82b-859e794b10bb

Screenshot 2024-04-18 at 14 14 59

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/services/signup-services/list-additional-data-info-service.js [hunk 1]
```diff
@@ -12,18 +11,27 @@ export const listAdditionalDataInfoService = async () => {
   return parseHttpResponse(httpResponse)
 }
 
+const adapt = (response) => {
+  response = [
+    ...response.slice(0, 3),
```
Comment (matheus-sunderhus) [inline src/services/signup-services/list-additional-data-info-service.js:16]: colocar em variáveis o que significa o 0 e o 3, bem como o id:10 ( criar variável para descrever esse valor 10 e o motivo de ser 10)

File: src/services/signup-services/list-additional-data-info-service.js [hunk 2]
```diff
@@ -12,18 +11,27 @@ export const listAdditionalDataInfoService = async () => {
...
+    { id: 10, key: 'Full Name', required: true, show: false },
+    ...response.slice(3)
+  ]
+
+  return response
+}
```
Comment (HerbertJulio) [inline src/services/signup-services/list-additional-data-info-service.js:22]: Sugiro colocar algo assim:

```suggestion
const adapt = (response) => {
   const positionFullName = 3;
   const fullName = { id: 10, key: 'Full Name', required: true, show: false };
   response.splice(indice, 0, fullName);

  return response
}
```

**Review by matheus-sunderhus — COMMENTED** [review]
minor changes required

File: src/services/signup-services/list-additional-data-info-service.js (hunk 2 above)
Comment (william-tome) [reply src/services/signup-services/list-additional-data-info-service.js]: Valeu pela sugestão. Nem tinha pensado nisso

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**